pages/jobs-form.tsx

```diff
@@ -8,45 +8,122 @@ export default function JobsForm() {
       </h1>
 
       <form
-        className="text-blue"
-        action="mailto:ryanlalchand@gmail.com"
+        className="p-4 flex-row"
+        action="https://mailto:contact@codeyoungstown.com?subject=New Job Listing"
```

Comment:
Does that actually work like that? I have no idea

Reply:
After doing some googling, I can't find a source that says to add `https://` to mailto links. Do you have something you were referencing?

Reply:
Just some Stack Overflow post as a hail mary. When I pushed this, the Vercel hosted version didn't display the Autofill Danger error